README.md

```diff
@@ -23,10 +23,7 @@ be found at [https://hexdocs.pm/ex_huobi](https://hexdocs.pm/ex_huobi).
 ## Margin Rest API Usage
 
 ```elixir
-config = %ExHuobi.Config{
-  api_key: System.get_env("HUOBI_API_KEY"),
-  api_secret: System.get_env("HUOBI_API_SECRET")
-}
+config = %{access_keys: ["XXX_HUOBI_API_KEY", "XXX_HUOBI_SECRET_KEY"]}
```

Comment:
@kimquy @quantd2 I seems don't understand we name it config here. It's should be access keys instead. A config is a map between key, value like original one.